corehq/apps/reports/standard/monitoring.py

```diff
@@ -202,32 +225,23 @@ def utc_now(self):
 
     @property
     def headers(self):
+
+        def make_column(title, help_text, num_days):
+            return DataTablesColumn(title, sort_type=DTSortType.NUMERIC,
+                                    help_text=help_text.format(num_days))
+
         columns = [DataTablesColumn(_("Users"))]
+
         for landmark in self.landmarks:
-            num_cases = DataTablesColumn(_("# Modified or Closed"), sort_type=DTSortType.NUMERIC,
-                help_text=_("The number of cases that have been modified between %d days ago and today.") % landmark.days
-            )
-            num_active = DataTablesColumn(_("# Active"), sort_type=DTSortType.NUMERIC,
-                help_text=_("The number of cases created or modified in the last %d days.") % landmark.days
-            )
-            num_closed = DataTablesColumn(_("# Closed"), sort_type=DTSortType.NUMERIC,
-                help_text=_("The number of cases that have been closed between %d days ago and today.") % landmark.days
-            )
-            proportion = DataTablesColumn(_("Proportion"), sort_type=DTSortType.NUMERIC,
-                help_text=_("The percentage of all recently active cases that were modified or closed in the last %d days.") % landmark.days
-            )
-            columns.append(DataTablesColumnGroup(_("Cases in Last %s Days") % landmark.days if landmark else _("Ever"),
-                num_cases,
-                num_active,
-                num_closed,
-                proportion
+            columns.append(DataTablesColumnGroup(
+                _("Cases in Last {} Days").format(landmark.days) if landmark else _("Ever"),
+                *[make_column(title, help_text, landmark.days)
+                  for title, help_text in self.landmark_columns]
             ))
-        columns.append(DataTablesColumn(_("# Active Cases"),
-            sort_type=DTSortType.NUMERIC,
-            help_text=_('Number of cases modified in the last %s days that are still open') % self.milestone.days))
-        columns.append(DataTablesColumn(_("# Inactive Cases"),
-            sort_type=DTSortType.NUMERIC,
-            help_text=_("Number of cases that are open but haven't been touched in the last %s days") % self.milestone.days))
```

Comment:
This whole thing just felt hard to read, so I pulled the relevant parts out as a standalone to also serve as a spec.